week-14/practices/12-context-practices/practice-1-README.md

```diff
@@ -87,24 +94,58 @@ can destructure:
   setCurrentSign: *function*
 }
 ```
+Now that we have our `currentSign`, we will need to grab some details about this
+sign. We have already created data for you to use, which is located in
+`src/data/horoscopes.js`. Take a second to familiarize how the data is setup,
+including how it's being exported.
+
+At the top of your `Detail.js` file, import the `horoscopeObj`. Next, inside
+the `Detail` component, create a new variable called `sign` and have it equal to
+the sign's object by keying into the `horoscopeObj`, using your `currentSign`
+state. Take a look at your `horoscopeObj` data again to help you see what we
+need.
+
+Below is the object we're trying to access if our `currentSign` is "Leo".
+
+```javascript
+ {
+    "name": "Leo",
+    "date": "Jul 23 - Aug 22",
+    "element":"Fire",
+    "traits": "Proud, Bold, Ambitious",
+    "backgroundImg":"https://www.horoscope.com/images-US/signs/profile-leo.png",
+    "match": "Libra"
+  }
+```
+
+Once you have successfully grabbed the object, update your `<img>` src, `<h1>`,
+`<h2>`, and `<h4>` elements with the according information of your `sign`
+object.
+
 
-With the `setCurrentSign` function, every time there is an `onClick` for the
-`<span>` element, we will `setCurrentSign` to the `sign`, so go ahead and
-implement that.
+If all was done correctly, when you refresh your browser, your `Detail`
+component should display all of your `currentSign`'s information.
 
-To confirm you have done this correctly, head back over to your browser and make
-sure your devTools => Component tab is open. Select your Navbar component and
-pay attention to your `hooks` section where your context is. As you click on
-each span, your `currentSign` should also update accordingly to the sign you
-have clicked on.
+Head back to your `HoroscopeContext.js` and update your default state to another
+sign! Does your `Detail` component also update?
 
+**HOORAY!**
 
-**Congratulations!**
+You have successfully created a dynamic value in your context provider and
+consumed that context to display your `currentSign`'s information.
 
-You have successfully used your context provider to dynamically change your
-value.
+What we have done are:
```

Comment:
**Line 137:** Would "What we have done is:" be better here?

Again, I'm not sure.

Reply:
I'm struggling with the grammar. I went with are because we used "have" and that's plural. But should it be related to "done", therefore "is" makes more sense? 👀

Reply:
I'm leaning towards "is" instead of "are" partially because it sounds more "right" and partially because I can't find "what we have done are" online